Add Supported Events Policy and update Reimbursement Policy (#4)
Speedcubing-Canada/docs-speedcubing-canada: public/src/supported-events-policy.md

<style type="text/css">
  a[href]:after { content : "" }
</style>

Version: 1.0
Updated: July 18, 2022

# Speedcubing Canada Supported Events Policy

Uses of the words “must”, “must not”, “should”, “should not” and “may” match [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).

All dollar amounts discussed are in Canadian dollars.

> Speedcubing Canada’s Board of Directors reserves the right tomodify this document with a majority of not less than two-thirds (2/3) of Speedcubing Canada’s Board of Directors vote in favour of the modification.

Reviewer: "tomodify"

> Speedcubing Canada’s Board of Directors reserves the right tomodify this document with a majority of not less than two-thirds (2/3) of Speedcubing Canada’s Board of Directors vote in favour of the modification.

Reviewer 2: ```suggestion
Speedcubing Canada’s Board of Directors reserves the right to modify this document if a majority of not less than two-thirds (2/3) of Speedcubing Canada’s Board of Directors vote in favour of the modification.
```

A Speedcubing Canada Supported Event is an event, usually an official World Cube Association (WCA) competition in Canada, that is organized by or in partnership with Speedcubing Canada under the following conditions:

## Recognition Process

- Written notice from Speedcubing Canada’s Board of Directors **must** be obtained to officially recognize a Speedcubing Canada Supported Event.
- Individuals seeking to run a Speedcubing Canada Supported Event **must** email [board@speedcubingcanada.org](mailto:board@speedcubingcanada.org) to request that their event be recognized as a Speedcubing Canada Supported Event.
  - The email **must** include a detailed, itemized budget for the event, including all expenses and sources of income.

>     - The budget _should_ be balanced and _should_ include a not insignificant margin to account for unexpected costs and circumstances.

Reviewer 2: Do you have a particular amount or percentage for the "not insignificant margin"?

  - If the email request is made by a Speedcubing Canada Director or Officer, the event will be recognized as a Speedcubing Canada Supported Event unless a Director raises an objection within 72 hours of the request.

>     - If an objection is raised by a Director, the event will be recognized as a Speedcubing Canada Supported Event if a majority of not less than two-thirds (2/3) of Speedcubing Canada’s Board of Directors vote in favour of recognition.

Reviewer 2: If a director requesting recognition, do they still get to vote?

Author: Probably makes sense to exclude them. I don't expect this to happen very often.

- Any and all reimbursements to individual(s) and/or group(s) for a Speedcubing Canada Supported Event **must** be made in accordance with the latest version of the [Speedcubing Canada Reimbursement Policy](https://www.speedcubingcanada.org/documents/reimbursement-policy.pdf).
- Speedcubing Canada’s Board of Directors reserves the right to request a full financial report for a Speedcubing Canada Supported Event from the individual(s) and/or group(s) who organized the event.
  - If requested, a full financial report **must** be returned to Speedcubing Canada’s Board of Directors within 30 days of the request.

## Legal Requirements and Insurance

- Any and all Speedcubing Canada Supported Events **must** follow all legal requirements of the jurisdiction in which in the event takes place.
- Any and all Speedcubing Canada Supported Events **must** be covered by a liability insurance policy (either provided by Speedcubing Canada directly or purchased by another party) with not less than $1 million coverage that names Speedcubing Canada as an insured party.

>   - If the insurance policy is not provided by Speedcubing Canada directly, proof of insurance **must** be emailed to [board@speedcubingcanada.org](mailto:board@speedcubingcanada.org).

Reviewer 2: ```suggestion
  - If the insurance policy is not provided by Speedcubing Canada directly, proof of insurance **must** be emailed to [board@speedcubingcanada.org](mailto:board@speedcubingcanada.org) before their event is recognized as a Speedcubing Canada Supported Event.
```
Or a similar qualification.

Author: In most cases Speedcubing Canada will purchase the insurance directly or the organizer will purchase the insurance via the venue (most municipalities in Ontario offer this).

I am comfortable with the insurance from the latter case being submitted "not less than two weeks before the first date of the event". If they don't provide proof of insurance, we can revoke recognition, or the more reasonable option would be purchasing ourselves at that point.

Anyways, hopefully this won't be a long term problem once we have ongoing coverage.

  - Speedcubing Canada’s Board of Directors reserves the right to request greater than $1 million coverage for a Speedcubing Canada Supported Event.

## Other

- Speedcubing Canada’s Board of Directors reserves the right to reject recognition of an event as a Speedcubing Canada Supported Event.

> - Speedcubing Canada’s Board of Directors reserves the right to ban specific individual(s) and/or group(s) who do not comply with Speedcubing Canada’s By-Laws and/or Policies from running Speedcubing Canada Supported Events for a fixed period of time or indefinitely.

Reviewer 2: ```suggestion
- Speedcubing Canada’s Board of Directors reserves the right to ban specific individual(s) and/or group(s) who do not comply with [Speedcubing Canada’s By-Laws](https://www.speedcubingcanada.org/documents/by-laws.pdf) and/or Policies from running Speedcubing Canada Supported Events for a fixed period of time or indefinitely.
```
Add link to by-laws.
